pkg/host/internal/network/network.go

```diff
@@ -286,3 +289,43 @@ func (n *network) SetDevlinkDeviceParam(pciAddr, paramName, value string) error
 	}
 	return nil
 }
+
+// EnableHwTcOffload makes sure that hw-tc-offload feature is enabled if device supports it
+func (n *network) EnableHwTcOffload(ifaceName string) error {
+	log.Log.V(2).Info("EnableHwTcOffload(): enable offloading", "device", ifaceName)
+	hwTcOffloadFeatureName := "hw-tc-offload"
+
+	knownFeatures, err := n.ethtoolLib.FeatureNames(ifaceName)
+	if err != nil {
+		log.Log.Error(err, "EnableHwTcOffload(): can't list supported features", "device", ifaceName)
+		return err
+	}
+	if _, isKnown := knownFeatures[hwTcOffloadFeatureName]; !isKnown {
+		log.Log.V(2).Info("EnableHwTcOffload(): can't enable feature, feature is not supported", "device", ifaceName)
```

Comment:
I think this can log with `V(0)`, or it can even return an error. 

It really depends on how you're going to use this function, but if the user set a flag somewhere to enable HwTcOffload, it is expected to work or to get an error somewhere. WDYT?
Same for L329

Reply:
agree, I fixed the log level